src/managers/StageInstanceManager.js

```diff
@@ -28,31 +28,30 @@ class StageInstanceManager extends BaseManager {
 
   /**
    * Options used to create a stage instance.
-   * @typedef {Object} CreateStageInstanceOptions
-   * @property {StageChannel|Snowflake} channel The stage channel whose instance is to be created
+   * @typedef {Object} StageInstanceCreateOptions
    * @property {string} topic The topic of the stage instance
    * @property {PrivacyLevel|number} [privacyLevel] The privacy level of the stage instance
    */
 
   /**
    * Creates a new stage instance.
-   * @param {CreateStageInstanceOptions} options The options to create the stage instance
+   * @param {StageChannel|Snowflake} channel The stage channel whose instance is to be created
```

Comment:
See my comment below:
```suggestion
   * @param {StageChannel|Snowflake} channel The stage channel to associate the created instance to
```